master/internal/task/allocation.go

```diff
@@ -86,6 +88,7 @@ type (
 	AllocationExited struct {
 		// userRequestedStop is when a container unexpectedly exits with 0.
 		UserRequestedStop bool
+		DaemonsKilled     bool
```

Comment:
so that's unused?

Reply:
forgot to delete this - is vestigial from removing stuff from trial.